includes/rest-api/class-sensei-rest-api-home-controller.php

```diff
@@ -0,0 +1,216 @@
+				],
+			],
+			'quick_links' => [
+				// TODO: Generate or hardcode in frontend.
+				'Courses'           => [
+					[
+						'title' => 'Getting Started Guide',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Create a Course',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Import a Course',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Reports',
+						'url'   => 'http://...',
+					],
+				],
+				'Settings'          => [
+					[
+						'title' => 'Email notifications',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Learning Mode',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'WooCommerce',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Content Drip',
+						'url'   => 'http://...',
+					],
+				],
+				'Advanced Features' => [
+					[
+						'title' => 'Interactive Blocks',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Groups & Cohorts',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Quizzes',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Integrations',
+						'url'   => 'http://...',
+					],
+				],
+			],
+			'help'        => [
+				// TODO: Generate or hardcode in frontend.
+			],
+			'guides'      => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro|interactive-blocks}.json.
+				'items'    => [
+					[
+						'title' => 'How to Sell Online Courses',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'How to Creating Video Courses',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'How to Choose the Right Hosting Provider',
+						'url'   => 'http://...',
+					],
+				],
+				'more_url' => 'http://senseilms.com/category/guides/',
+			],
+			'news'        => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro|interactive-blocks}.json.
+				'items'    => [
+					[
+						'title' => 'Introducing Interactive Videos For WordPress',
+						'date'  => '2022-08-31T21:41:38',
+						'url'   => 'http://senseilms.com/inroducing-interactive-videos/',
+					],
+					[
+						'title' => 'New Block Visibility, Scheduled Content, and Group Features',
+						'date'  => '2022-08-09T20:51:51',
+						'url'   => 'http://senseilms.com/conditional-content/',
+					],
+				],
+				'more_url' => 'https://senseilms.com/blog/',
+			],
+			'extensions'  => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro}.json.
+				[
+					'title'        => 'Sensei LMS Post to Course Creator',
+					'image'        => 'http://senseilms.com/wp-content/uploads/2022/02/sensei-post-to-course-80x80.png',
+					'description'  => 'Turn your blog posts into online courses.',
+					'price'        => 0,
+					'is_featured'  => false, // To be used to differentiate between featured and not featured extensions.
```

Comment:
Featured and non-featured extensions are all returned in a single collection but can be differentiated by `is_featured` property.

Reply:
Unless I'm missing something obvious, are we going to have featured extensions?

I remember that we had those on the extensions, but I'm not 100% sure about the new Sensei Home. I think Sensei Pro is the only extension that we really want to feature more... :thinking:

Reply:
Hey, just as FYI, I just got a reply from Ronnie confirming that we won't need featured extensions support: p6rkRX-4so-p2#comment-4848

Reply:
Ok, thanks for looking into that! Removed here 7a7dfd105

And added support for the promotional banner in a very simple way here 1a725d3dc
(I have added more details/doubts about the promotional banner here)